soc/arm/silabs_exx32/Kconfig

```diff
@@ -340,4 +340,11 @@ config SOC_GECKO_USE_RAIL
 	  hardware. This option enable the proper set of features to allow to properly compile
 	  with the RAIL blob.
 
+config SOC_GECKO_CUSTOM_RADIO_PHY
+	bool "Use RAIL for custom radio phy packet sending and receiving"
```

Comment:
Or `SOC_GECKO_CUSTOM_RADIO_PHY` should `depends on SOC_GECKO_USE_RAIL`?

Reply:
I would go with the following:
`select SOC_GECKO_USE_RAIL`
`depends on CONFIG_SOC_GECKO_HAS_RADIO`
With this it will be only selectable if the boards supports radio, and if it does, it will automatically enable USE_RAIL.
Would this be okay?